### PR TITLE
Feature/create layout navbar

### PR DESCRIPTION
**Pull request by devDrodva**
create layout navbar

### CODE REVIEW

File: src/components/Navbar/styles.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components'
+
+export const StyledNav = styled.nav`
+  width: 100%;
+  height: 50px;
+  margin: 0 auto;
+  display: grid;
```
Comment (sgbenav) [inline src/components/Navbar/styles.js:7]: Using grid in the navbar seems to be overkill. Can we use a display flex?

**Comment by devDrodva** [comment]
Template

File: src/components/Navbar/styles.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  align-content: center;
+  background-color: #fff;
+`
+
+export const StyledLogo = styled.h1`
+  margin-left: 250px;
```
Comment (sgbenav) [inline src/components/Navbar/styles.js:13]: I think there is a better approach. Can we setup the space within the navbar as a padding in the nav element so both sides left and right will be equal. Something like: padding: 0 100px;

File: src/components/Navbar/index.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { StyledNav, StyledLogo } from './styles'
+
+const Nav = () => {
```
Comment (sgbenav) [inline src/components/Navbar/index.js:4]: Great work using styled components

File: src/components/CardList/styles.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components'
+
+const StyledCardList = styled.div`
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(4, 150px);
```
Comment (sgbenav) [inline src/components/CardList/styles.js:6]: Is this responsive?

File: src/assets/logo.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg aria-hidden="true" focusable="false" data-prefix="fab" data-icon="artstation" class="svg-inline--fa fa-artstation fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="currentColor" d="M2 377.4l43 74.3A51.35 51.35 0 0 0 90.9 480h285.4l-59.2-102.6zM501.8 350L335.6 59.3A51.38 51.38 0 0 0 290.2 32h-88.4l257.3 447.6 40.7-70.5c1.9-3.2 21-29.7 2-59.1zM275 304.5l-115.5-200L44 304.5z"></path></svg>
```
Comment (sgbenav) [inline src/assets/logo.svg:1]: This is missing and empty line at the end

File: src/App.js [hunk 6]
```diff
@@ -1,7 +1,14 @@
 import React from 'react'
+import Nav from './components/Navbar/index'
+import Main from './components/Main/index'
 
 function App() {
-  return <div className='App'>Hola</div>
+  return (
+    <div className='App'>
```
Comment (sgbenav) [inline src/App.js:7]: Are we using a class named App in the css?

File: src/components/CardList/index.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import StyledCardList from './styles'
+import Card from '../Card/index'
+
+const CardList = () => {
+  return (
+    <StyledCardList>
```
Comment (sgbenav) [inline src/components/CardList/index.js:7]: No CardList component or Card component was specified in the ticket. This component will be done in the next story and with other name. Please, code only the AC (Acceptance Criteria). Place a sigle div that can be displayed as a card inside the main.

File: src/components/Navbar/styles.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+  height: 50px;
+  margin: 0 auto;
+  display: Flex;
+  align-content: center;
+  justify-content: space-between;
+  background-color: #fff;
+`
+
+export const StyledLogo = styled.h1`
+  padding: 0 100px;
```
Comment (sgbenav) [inline src/components/Navbar/styles.js:14]: Can we move the padding to the navbar?

File: src/components/Main/styles.js [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+`
+export const StyledCard = styled.div`
+  width: 160px;
+  height: 300px;
+  background-color: #fff;
+`
+export const StyledWrapperCards = styled.div`
+  width: 70%;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(150px, 2fr));
```
Comment (sgbenav) [inline src/components/Main/styles.js:18]: looks better

File: README.md [hunk 10]
```diff
@@ -6,7 +6,16 @@
 ```
 - src
 - - - - components
-      |- Cart
+      |- CardList
```
Comment (sgbenav) [inline README.md:9]: Update the documentation if the component no longer exist

File: src/App.js [hunk 11]
```diff
@@ -1,7 +1,14 @@
 import React from 'react'
+import Nav from './components/Navbar/index'
+import Main from './components/Main/index'
 
 function App() {
-  return <div className='App'>Hola</div>
+  return (
+    <div>
```
Comment (sgbenav) [inline src/App.js:7]: Is this div necessary or can we use a `<>` to avoid non-semantic tags

**Comment by devDrodva** [comment]
Template

**Review by sgbenav — APPROVED** [review]
Approved

**Pull request merged**